data/competencies.yml

```
@@ -511,6 +521,8 @@
     - Publishes a new origami component that uses other components
     - Implements a CDN / gateway that routes to a suite of underlying microservices
     - Designs and implements a build pipeline
+    - creates/modifies/improves Kubernetes operators
+    - creates/modifies/improves Docker files/images
```

We're not 100% sure of the wording of these. Do they demonstrate working with systems that involve multiple independent technical parts? I think this might be a lack of expertise on my part, would be good to chat with the person who suggested these so I can help reword
```suggestion
    - Creates, modifies, or improves Kubernetes operators
    - Creates, modifies, or improves Docker files/images
```